stdlib/Array.dora

```diff
@@ -1,17 +1,17 @@
-internal class Array<T>(len: Int) {
+internal class Array[T](len: Int) {
 
   internal fun length() -> Int;
   internal fun get(idx: Int) -> T;
   internal fun set(idx: Int, val: T);
 
-  fun contains(value: T /* : Equals*/) -> Bool {
+  fun contains[T /*: Equals*/](value: T) -> Bool {
     var i = 0;
 
     while i < self.length() {
-      let x = self[i];
-      if /*x.equals(value) ||*/ x === value {
-        return true;
-      }
+      let x = self.get(i);
+      //if x.equals(value) || x === value { // TypesIncompatible("T", "T")
```

Comment:
If we can't express this function right now, we should probably remove it until we can.

Reply:
Ah, weird, I'm not even sure why this ended up in this commit... I'll fix it! Thanks!